harpy/downsample.py

```
@@ -3,8 +3,7 @@
 import os
 import re
 import sys
-import yaml
-import shutil
+from turtle import down
```

Ruff (0.11.9)

`turtle.down` imported but unused

Remove unused import: `turtle.down`

(F401)